pkg/image/api/types.go

```diff
@@ -60,6 +77,30 @@ type ImageLayer struct {
 	Size int64
 }
 
+// SignatureValidationResult holds a result of signature validation.
+type SignatureValidationResult string
+
+// ImageSignature holds a signature of an image.
+type ImageSignature struct {
```

Comment:
I should usually be able to get a `SignerSubject` and `CertificateSubject` right?  That would tell me which signing key was used and the `CertificateSubject` would let me know who confirmed the key.